lib/Dredd.js

```diff
@@ -1,21 +1,89 @@
 const async = require('async');
-const clone = require('clone');
-const fs = require('fs');
-const request = require('request');
-const url = require('url');
 const parse = require('dredd-transactions/parse');
 const compile = require('dredd-transactions/compile');
 
 const configureReporters = require('./configureReporters');
 const handleRuntimeProblems = require('./handleRuntimeProblems');
 const resolveLocations = require('./resolveLocations');
+const readLocation = require('./readLocation');
 const resolveModule = require('./resolveModule');
 const logger = require('./logger');
 const TransactionRunner = require('./TransactionRunner');
 const { applyConfiguration } = require('./configuration');
 
 
-const FILE_DOWNLOAD_TIMEOUT = 5000;
+function prefixErrors(decoratedCallback, prefix) {
```

Comment:
I couldn't come up with a nicer way how to prefix errors with context than decorating the callback like this. I might be just missing a good idea or a best practise. But it's gonna disappear with async/await/promises anyway.